<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Alat Kesehatan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kode Alat Kesehatan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtKodeAlkes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nama Alat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNamaAlkes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridAlkes">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>541</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="btnCari">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cari</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Id Alat Kesehatan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Harga Alat Kesehatan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHargaAlkes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIdAlkes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>118</width>
      <height>63</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Salsabila Niarno</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>200511152</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R2</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btnSimpan">
     <property name="styleSheet">
      <string notr="true">#btnSimpan{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 255);
}</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="styleSheet">
      <string notr="true">#btnClear{
	background-color: rgb(0, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="styleSheet">
      <string notr="true">#btnHapus{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}
</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
